package.json

```diff
@@ -39,6 +39,10 @@
     "docs/src/rules/*.md": [
       "node tools/fetch-docs-links.js",
       "git add docs/src/_data/further_reading_links.json"
+    ],
+    "**/*.svg": [
+      "npx svgo -r",
```

Comment:
Might include this `--multipass ` argument which can further optimize the images:
```
--multipass                Pass over SVGs multiple times to ensure all optimizations are applied
```

Reply:
done.